docs/bugpattern/MissingOverride.md

```diff
@@ -4,5 +4,8 @@ overriding a superclass method, a class method implementing an interface method,
 and an interface method respecifying a superinterface method.
 
 Exception: `@Override` may be omitted when the parent method is `@Deprecated`.
+If the option `-XepOpt:MissingOverride:IgnoreInterfaceOverrides=true`
```

Comment:
Fwiw, https://errorprone.info/docs/flags#pass-additional-info-to-bugcheckers calls them _flags_ rather than _options_.

Reply:
Thanks, fixed!